Add new values for `TaxId`

remi-stripe opened the pull request:
Codegen for openapi 81375b4

r? @ob-stripe 
cc @stripe/api-libraries

ob-stripe commented on types/2019-12-03/Issuing/Disputes.d.ts:
```diff
@@ -107,9 +107,9 @@ declare module 'stripe' {
+         * The reason for the dispute. One of `fraudulent` or `other`.
          */
-        reason: DisputeCreateParams.Reason;
+        reason: string;
```
Let's hold onto this release until we fix this unnecessary breaking change.

remi-stripe commented:
For completeness, most of it was done by a few regexp and then some manual fixes:

First replace all top-level CRUD methods

Input:
```
.*(Get|Delete|Update)(Async)*)\(string [a-zA-Z]+Id,( [^s].*)
(.*{.*)
(.*(Get|Delete|Update)Entity(Async)*)\([a-zA-Z]+Id,
```
Replaced by:
```
$1(string id,$4
$5
$6(id,
```

Then replace nested RUD methods
```
(.*(Get|Delete|Update)(Async)*)\(string [a-zA-Z]+Id, string [a-zA-Z]+Id,( .*)
(.*{.*)
(.*(Get|Delete|Update)NestedEntity(Async)*)\([a-zA-Z]+Id, [a-zA-Z]*Id,
```
Replaced by
```
$1(string parentId, string id,$4
$5
$6(parentId, id,
```

Then replace nested Create
```
(.*(Create)(Async)*)\(string [a-zA-Z]+Id,( [^s].*)
(.*{.*)
(.*(Create)NestedEntity(Async)*)\([a-zA-Z]+Id,
```
Replaced by
```
$1(string parentId,$4
$5
$6(parentId,
```

Then replace nested List (async and autopaging)
```
(.*(List)(Async|Autopaging)*)\(string [a-zA-Z]+Id,( [^s].*)
(.*{.*)
(.*(List)NestedEntities(Async|Autopaging)*)\([a-zA-Z]+Id,
```
Replaced by
```
$1(string parentId,$4
$5
$6(parentId,
```
